server/src/routes/token.ts

```diff
@@ -1,6 +1,7 @@
 import { Request, RequestHandler, Response } from 'express';
 import { jwt } from 'twilio';
 import { TwilioCredentials } from '../common/types';
+import { error } from '../utils/log';
```

Comment:
Where do we use this? I see an `import` but the GitHub diff doesn't show any uses.